.eslintrc.json

```diff
@@ -22,8 +23,37 @@
     "project": "./tsconfig.eslint.json"
   },
   "rules": {
-    "no-var": "error",
+    // TS
     "@typescript-eslint/explicit-function-return-type": "off",
-    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }]
+    "@typescript-eslint/no-unused-vars": [
+      "warn",
+      { "argsIgnorePattern": "^_" }
+    ],
+    "@typescript-eslint/no-use-before-define": "off",
+
+    // ESLint
+    "max-lines-per-function": "off",
+    "no-restricted-imports": [
+      "error",
+      {
+        "patterns": ["@typescript-eslint/experimental-utils/dist/*"]
+      }
+    ],
+
+    // Import
+    "import/order": [
+      "warn",
+      {
+        "groups": ["builtin", "external", "parent", "sibling", "index"],
+        "pathGroups": [
+          { "pattern": "@typescript-eslint/**", "group": "external" }
```

Comment:
Yes, you are right! But `eslint-plugin-import` was assuming this was an internal, not sure why, so I had to add this. Could be a bug in the plugin? I couldn't find any reason for assuming `@typescript-eslint/**` was internal.

Reply:
Still having the issue after updating all the dependencies to latest?

If not, let's remove it.
Otherwise, I think it would be best to create a small reproduction and issue a bug report upstream

Reply:
Yes, actually I started getting that weird sort **after** upgrading to the latest `eslint-plugin-import` version. I see few issues open in their repo regarding the 2.23 version. I'm gonna try downgrading the package.

Reply:
I just figured it out: [`import type`s are sorted after regular imports since 2.23.0](https://github.com/benmosher/eslint-plugin-import/pull/2021)

I'm gonna remove my custom config for including the package as internal since that wasn't the problem.